README.md

```diff
@@ -7,4 +7,16 @@
 
 ### AWS Architecture Diagram
 
-![AWS Architecture Diagram](images/architecture_diagram.png)
\ No newline at end of file
+![AWS Architecture Diagram](images/architecture_diagram.png)
+
+
+### Terraform Configuration:
+The `terraform` folder contains all files for terraform infrastructure:
+- `provider.tf` - defines the AWS region.
+- `s3_bucket.tf` - provisioning the AWS S3 bucket for long-term data storage (data older than 24hrs).
+  
+One script for each of the 3 ECRs utilised in this project:
+- `dashboard_ecr.tf`
+- `etl_ecr.tf`
+- `data_backup_ecr.tf`
+- Each ECR has a lifecycle policy which keeps only the 30 most recently pushed images.
```

Comment:
Nice concise explanations!

Reply:
Could maybe embellish a little more on what each of the ECRs do? 'dashboard_ecr.tf' is a little vague without explaining that its for a data dashboard where users can see daily trends.

Reply:
@bentrzcinski good shout, will add descriptions for each of those